packages/mobile/src/store/products/reducer.ts

```diff
@@ -1,15 +1,15 @@
 import { createReducer } from '@reduxjs/toolkit';
+import { AllProductsDto } from '@vse-bude/shared';
 import { DataStatus } from '~/common/enums/enums';
-import { ProductDto } from '@vse-bude/shared';
 import { loadProducts } from './actions';
 
 type InitialState = {
-  products: ProductDto[] | [];
+  products: AllProductsDto;
   dataStatus: DataStatus;
 };
 
 const initialState: InitialState = {
-  products: [],
+  products: { items: [], count: 0 },
```

Comment:
It will brake a home page . Expected array from products

Reply:
it won't, for products in selector select products.items, so it would be array as expected